src/algos/value_iteration.jl

```diff
@@ -1,5 +1,17 @@
 using Optim
 
+
+"""
+#TODO: Is evaluate_policy the best name for this? Seems that we are evaluating the value function, or evaluating a current decision rule.
```

Comment:
But it's quite literally an evaluation of the value function, not an evaluation of the policy (which would be something like `dr(states, shocks, ...)`). 

I'm also confused by the fact that there another function in the module called `update_value`. Since both functions are referring to the value function, and both are evaluating that value function in some way, why would one refer to a "policy" and the other refer to a "value"?